inventory/sample-ha.aws.example.com.d/inventory/group_vars/all.yml

```diff
@@ -0,0 +1,56 @@
+# 'hosting_infrastructure' is used to drive the correct behavior based
+# on the hosting infrastructure, cloud provider, etc. Valid values are:
+# - 'openstack'
+# - 'aws' (Coming Soon)
+# - 'azure' (Coming Soon)
+hosting_infrastructure: aws
+
+# AWS Provisioning Variables
+# use the "-e" option to set "aws_ssh_public_key"
+aws_image_name: <ec2 AMI>
+aws_num_masters: 3
+aws_num_nodes: 5
+aws_num_infra: 3
+aws_access_key: <Access key ID>
+aws_secret_key: <Secret access key>
+aws_region: <AWS Region>
+aws_key_name: <Key pair name>
+aws_subnet: <Subnet ID>
+# If aws_create_vpc is 'true', aws_subnet variable will be ignored
+aws_create_vpc: <true|false>
+# These are the security groups created when aws_create_vpc is 'true'. Modify accordingly to your environment in case using existing VPC and SGs
+aws_master_sgroups: ['labs-ssh', 'labs-ocp-master', 'labs-ocp-node']
+aws_infra_sgroups: ['labs-ssh', 'labs-ocp-infra', 'labs-ocp-node']
+aws_node_sgroups: ['labs-ssh', 'labs-ocp-node']
+
+# Cluster variables
+env_id: "sample"
+public_dns_domain: "sre.tests.com"
+public_dns_nameservers:
+- 8.8.8.8
+ha_mode: true
+
+
+# Dockr configuration variables
+#docker_volume_size: "5"
```

Comment:
Why commented out? We need to source the size from somewhere?

Reply:
This is initially only required by OSP:
```
#~ grep -r docker_volume_size . 2>/dev/null | grep -v inventory
./playbooks/dns/roles/openshift-ansible-contrib/playbooks/provisioning/openstack/README.md:The `docker_volume_size` is the default Docker volume size the servers will use.
./playbooks/dns/roles/openshift-ansible-contrib/playbooks/provisioning/openstack/README.md:for master) and change its value. `docker_volume_size` must stay defined as it is
./playbooks/dns/roles/openshift-ansible-contrib/playbooks/provisioning/openstack/stack_params.yaml:master_volume_size: "{{ docker_master_volume_size | default(docker_volume_size) }}"
./playbooks/dns/roles/openshift-ansible-contrib/playbooks/provisioning/openstack/stack_params.yaml:infra_volume_size: "{{ docker_infra_volume_size | default(docker_volume_size) }}"
./playbooks/dns/roles/openshift-ansible-contrib/playbooks/provisioning/openstack/stack_params.yaml:node_volume_size: "{{ docker_node_volume_size | default(docker_volume_size) }}"
./playbooks/nagios/roles/openshift-ansible-contrib/playbooks/provisioning/openstack/README.md:The `docker_volume_size` is the default Docker volume size the servers will use.
./playbooks/nagios/roles/openshift-ansible-contrib/playbooks/provisioning/openstack/README.md:for master) and change its value. `docker_volume_size` must stay defined as it is
./playbooks/nagios/roles/openshift-ansible-contrib/playbooks/provisioning/openstack/stack_params.yaml:master_volume_size: "{{ docker_master_volume_size | default(docker_volume_size) }}"
./playbooks/nagios/roles/openshift-ansible-contrib/playbooks/provisioning/openstack/stack_params.yaml:infra_volume_size: "{{ docker_infra_volume_size | default(docker_volume_size) }}"
./playbooks/nagios/roles/openshift-ansible-contrib/playbooks/provisioning/openstack/stack_params.yaml:node_volume_size: "{{ docker_node_volume_size | default(docker_volume_size) }}"
./playbooks/openshift/provision-openstack.yml:    master_volume_size: "{{ docker_volume_size }}"
./playbooks/openshift/provision-openstack.yml:    node_volume_size: "{{ docker_volume_size }}"
./playbooks/openshift/provision-openstack.yml:    infra_volume_size: "{{ docker_volume_size }}"
./playbooks/openshift/roles/openshift-ansible-contrib/playbooks/provisioning/openstack/README.md:The `docker_volume_size` is the default Docker volume size the servers will use.
./playbooks/openshift/roles/openshift-ansible-contrib/playbooks/provisioning/openstack/README.md:for master) and change its value. `docker_volume_size` must stay defined as it is
./playbooks/openshift/roles/openshift-ansible-contrib/playbooks/provisioning/openstack/stack_params.yaml:master_volume_size: "{{ docker_master_volume_size | default(docker_volume_size) }}"
./playbooks/openshift/roles/openshift-ansible-contrib/playbooks/provisioning/openstack/stack_params.yaml:infra_volume_size: "{{ docker_infra_volume_size | default(docker_volume_size) }}"
./playbooks/openshift/roles/openshift-ansible-contrib/playbooks/provisioning/openstack/stack_params.yaml:node_volume_size: "{{ docker_node_volume_size | default(docker_volume_size) }}"
./playbooks/roles/openshift-ansible-contrib/playbooks/provisioning/openstack/README.md:The `docker_volume_size` is the default Docker volume size the servers will use.
./playbooks/roles/openshift-ansible-contrib/playbooks/provisioning/openstack/README.md:for master) and change its value. `docker_volume_size` must stay defined as it is
./playbooks/roles/openshift-ansible-contrib/playbooks/provisioning/openstack/stack_params.yaml:master_volume_size: "{{ docker_master_volume_size | default(docker_volume_size) }}"
./playbooks/roles/openshift-ansible-contrib/playbooks/provisioning/openstack/stack_params.yaml:infra_volume_size: "{{ docker_infra_volume_size | default(docker_volume_size) }}"
./playbooks/roles/openshift-ansible-contrib/playbooks/provisioning/openstack/stack_params.yaml:node_volume_size: "{{ docker_node_volume_size | default(docker_volume_size) }}"
./roles/openshift-ansible-contrib/playbooks/provisioning/openstack/README.md:The `docker_volume_size` is the default Docker volume size the servers will use.
./roles/openshift-ansible-contrib/playbooks/provisioning/openstack/README.md:for master) and change its value. `docker_volume_size` must stay defined as it is
./roles/openshift-ansible-contrib/playbooks/provisioning/openstack/stack_params.yaml:master_volume_size: "{{ docker_master_volume_size | default(docker_volume_size) }}"
./roles/openshift-ansible-contrib/playbooks/provisioning/openstack/stack_params.yaml:infra_volume_size: "{{ docker_infra_volume_size | default(docker_volume_size) }}"
./roles/openshift-ansible-contrib/playbooks/provisioning/openstack/stack_params.yaml:node_volume_size: "{{ docker_node_volume_size | default(docker_volume_size) }}"```

Reply:
Actually, as you'll see from a later comment (see further down), there's a hardcoded size of 40 right now - that needs to be replaced with this variable.